### ISSUE TITLE
Default values for command line args should not overwrite config file

### ISSUE THREAD

**Issue by dowd-a** [FCP-INDI/C-PAC#894]
I just ran CPAC with the default pipeline config (I copied the one it automatically uses for the participant setting), with one exception. I changed the field removeWorkingDir from true to false and added it using the --pipeline_file command line argument. So CPAC should not have removed the working directory when it finished running. However, it still did remove it. When I checked the pipeline config file that CPAC created based on the one I gave it, the removeWorkingDir flag had been flipped back to true.

I tried running the exact same command again. I included the same pipeline config file with the removeWorkingDir field set to false, but this time I also included the --save_working_dir flag. And the working directory was not deleted.

Basically it seems like the default values of the unused flags are overwriting the pipeline config file given by the user.

**Comment by sgiavasis** [comment]
This has been resolved, and will be in 1.4.2!
